src/Chapter2.hs

```diff
@@ -336,7 +337,11 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start stop xs
+  | start < 0 || stop < 0 = [] -- return an empty list when given numbers are negative.
+  | start > stop          = [] -- >>> subList 3 0 [True, False, False, True, False] = []
```

Comment:
Nice catch!
This could be spatted in the next case, but it is better to have this check separately for performance!
Very cool 😎